app/assets/javascripts/application.js.erb

```diff
@@ -230,6 +230,7 @@ var HelperModule = (function(){
   return helpers;
 })();
 
+// initialize code markup in step descriptions and text results
```

Comment:
Please don't :feelsgood: me. Anyway, can you simply put `initialize code markup in rich text fields`, it makes more sense once we expand rich text functionality to more fields in application.